config/schedule.rb

```diff
@@ -2,10 +2,6 @@
 
 set :output, '/dev/null'
 
-every 1.day do
```

Comment:
Why do we remove this now? I wonder could we make `find:all` idempotent and then still run that daily?